### PR TITLE
Fixed video device initialization

### PR DESCRIPTION
**Pull request by s40in**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-# VMW Add-ons v1.2.0
+# VMW Add-ons v1.2.3
+
+This is my repo for test my fixes.
```
Comment (diversys) [inline README.md:3]: Shouldn't be needed.

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
 objects*/
+_build_video.sh
```
Comment (diversys) [inline .gitignore:2]: Looks like a leftover?

File: README.md (hunk 1 above)
Comment (s40in) [reply README.md]: Removed

File: .gitignore (hunk 2 above)
Comment (s40in) [reply .gitignore]: Removed

**Pull request merged**